bedrock/foundation/templates/foundation/annualreport/2019/index.html

```diff
@@ -542,25 +544,23 @@ <h2 class="mzp-c-card-feature-title"><span class="highlight green">The internet
       </section>
     </div>
 
-    <!-- Feature Card -->
-    <div class="mzp-l-content no-js-hidden">
-      <section id="victory-has-a-ring-to-it" data-modal-id="victory-has-a-ring-to-it" class="mzp-c-card-feature mzp-has-aspect-16-9 mzp-l-card-feature-left-third has-modal">
-        <div class="mzp-c-card-feature-media-wrapper">
-          <div class="mzp-c-card-feature-media">
-            {{ lazy_img(image_url='img/foundation/annualreport/2019/victory-ring.jpg', placeholder_url='img/foundation/annualreport/2019/spacer.gif', include_highres_image=True, optional_attributes={'alt': '', 'width': '930', 'height': '523', 'class': 'mzp-c-card-image'}) }}
-          </div>
-        </div>
-        <div class="mzp-c-card-feature-content">
-          <div class="mzp-c-card-feature-content-container">
-            <div class="mzp-c-card-tag">Taking on Big Tech</div>
-            <h2 class="mzp-c-card-feature-title">Victory has a Ring to&nbsp;it </h2>
-            <button class="mzp-c-button mzp-t-secondary">
-              Our campaign for real home security
-            </button>
-          </div>
-        </div>
-      </section>
-    </div>
+<div class="has-modal" data-modal-id="victory-has-a-ring-to-it">
+  {% call split(
+    id='victory-has-a-ring-to-it',
+    image_url='img/foundation/annualreport/2019/victory-ring.jpg',
+    include_highres_image=True,
+    media_after=False,
+    media_class='mzp-l-split-h-center',
+    theme_class='mzp-l-split-body-narrow',
+    block_class='mzp-t-split-nospace'
+    ) %}
+    <span>Taking on Big Tech</span>
+    <h2>Victory has a Ring to it</h2>
+    <button type=button class="mzp-c-button mzp-t-secondary">
```

Comment:
Nit: attribute values in HTML should be double-quoted
```suggestion
    <button type="button" class="mzp-c-button mzp-t-secondary">
```
Same on the other buttons, too.